dev/diffs/4.0.0.diff

```diff
@@ -71,62 +94,85 @@ index 466e4cf8131..798f118464c 100644
    /**
     * Initialize extensions specified in [[StaticSQLConf]]. The classes will be applied to the
     * extensions passed into this function.
-@@ -1385,7 +1393,8 @@ object SparkSession extends Logging {
+@@ -1049,9 +1066,10 @@ object SparkSession extends SparkSessionCompanion with Logging {
+   private def applyExtensions(
+       sparkContext: SparkContext,
        extensions: SparkSessionExtensions): SparkSessionExtensions = {
-     val extensionConfClassNames = sparkContext.getConf.get(StaticSQLConf.SPARK_SESSION_EXTENSIONS)
+-    val extensionConfClassNames = sparkContext.conf.get(StaticSQLConf.SPARK_SESSION_EXTENSIONS)
++    val extensionConfClassNames = sparkContext.getConf.get(StaticSQLConf.SPARK_SESSION_EXTENSIONS)
```

Comment:
So looks this diff is changing from `conf` to `getConf`
Any particular reasons?

Reply:
I think this is an accidental change. Will undo.